lib/pure/strutils.nim

```diff
@@ -89,6 +89,9 @@ const
     ## All the characters that count as whitespace (space, tab, vertical tab,
     ## carriage return, new line, form feed).
 
+  Punctuation* = {'\33'..'\47', '\58'..'\64', '\91'..'\96', '\123'..'\126'}
+    ## All the characters that count as punctuation (comma, colon, dash etc).
```

Comment:
can you provide in this PR a link to what some other languages do here?

Reply:
For example:
- https://en.cppreference.com/w/c/string/byte/ispunct
- https://docs.python.org/3/library/string.html?highlight=string#string.punctuation